pytest_rpc/helpers.py

```diff
@@ -501,24 +502,16 @@ def parse_swift_recon(recon_out):
     """
 
     lines = recon_out.splitlines()
+    delimiter_regex = re.compile('={20,}')
```

Comment:
I would prefer to assume a full 80n character delimiter (79 `=` and a newline) to ensure that we are not caught by some other type of output.
Remote possibility:
```
Problem with host 3 <====================
```